# Behaviour model
string model         # Base58 
# Task
string objective     # Base58
# Operational token
string token         # address
# Execution cost
uint32 cost          # int
# Validator network address
string validator     # address
# Validator network fee
uint32 validatorFee  # int 
# Deadline block number
uint32 deadline      # int
# Order nonce value
uint8[] nonce        # hex 
# Order signature
uint8[] signature    # hex
